Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -76,16 +124,52 @@ private AtlassianUser ToAtlassianUser(JiraUser user, long instanceId)
             };
         }
 
-        private void UpdateDbUser(AtlassianUser dbUser, AtlassianUser updatedUser)
+        private List<AtlassianWorklog> GetWorklogsFromIssueWorklogs(List<JiraIssue> issueWorklog, List<AtlassianUser> users, long instanceId)
+        {
+            var worklogs = new List<AtlassianWorklog>();
+
+            foreach (var issue in issueWorklog)
+            {
+                foreach (var worklog in issue.fields.worklog.worklogs)
+                {
+                    var user = users.SingleOrDefault(p => p.Key == worklog.author.name);
+
+                    worklogs.Add(new AtlassianWorklog()
+                    {
+                        JiraWorklogId = Int64.Parse(worklog.id),
```

Comment:
I think you can change the type of worklog.id to long or int. so you won't need this cast